css/index.css

```diff
@@ -143,4 +174,108 @@ footer nav {
 footer nav a {
     color: white;
     text-decoration: none;
+}
+
+.middle-content.boxes.box Box1{
+    background-color: teal
+}
+
+.box1{
+    background-color: gold
+}
+.box Box3{
+    background-color: cadetblue;
+}
+.box Box4{
+    background-color: coral;
+}
+.box Box5{
+    background-color: crimson;
+}
+.box Box6{
+    background-color: forestgreen;
+}
+.box Box7{
+    background-color: darkorchid;
+}
+.box Box8{
+    background-color: hotpink;
+}
+.box Box9{
+    background-color: indigo;
+}
+.box Box10{
+    background-color: dodgerblue; 
+}
+
+.top-about{
+    margin-top: 5px;
+    margin-bottom: 5px;
+    padding: 30px;
+    border-bottom: 1px dashed black;
+}
+
+.pg2content{
+    display: flex;
+    flex-direction: column;
+    padding: 10px;
+}
+
+.firsthalf{
+    display: flex;
+    flex-direction: row;
+    justify-content: space-evenly;
+}
+
+.secondhalf{
+    display: flex;
+    flex-direction: row;
+    justify-content: space-evenly;
+}
+
+.partone{
+    display: flex;
+    flex-direction: column;
+    width: 45%;
+    border-bottom: 1px dashed black;
+    margin-bottom: 30px;
+}
+
+.parttwo{
+    display: flex;
+    flex-direction: column;
+    width: 45%;
+    border-bottom: 1px dashed black;
+    margin-bottom: 30px;
+}
+
+.partthree{
+    display: flex;
+    flex-direction: column;
+    width: 45%;
+    border-bottom: 1px dashed black;
+    margin-bottom: 30px;
+}
+
+.partfour{
+    display: flex;
+    flex-direction: column;
+    width: 45%;
+    border-bottom: 1px dashed black;
+    margin-bottom: 30px;
+}
+
```

Comment:
these can be combined to 1 class to make it more non recursive

Reply:
Not sure what you meant with this comment? Did you mean I could have done this with the parent class and it would have applied across the board?